ibr/ibr-sdk/src/IBRObject.js

```diff
@@ -137,13 +124,25 @@ Object.assign( IBRObject.prototype, {
     return this.coordinates;
   },
 
+  /**
+   * Get blocking grid.
+   * @return {BlockingGrid} Blocking grid of the IBRObject.
+   */
+  getBlockingGrid: function() {
+    return this.blockingGrid;
+  },
+
   /**
    * Convert IBRObject object to JSON format.
    * @return {JSONObject} json format of IBRObject object.
    */
   toJson: function() {
     const json = {};
-    json.blocking_grid = this.blockingGrid;
+    if (this.hasBlockingGrid) {
+      json.blocking_grid = this.blockingGrid.toJson();
+    } else {
+      json.blocking_grid = this.blockingGrid;
```

Comment:
does this else need to be here? is it possible to have something in blockingGrid but hasBlockingGrid is false?

Reply:
Changed to set to null when hasBlockingGrid is false